src/AutorestSwift/View Models/ServiceClientFileViewModel.swift

```diff
@@ -35,18 +35,46 @@ struct ServiceClientFileViewModel {
     let apiVersionName: String
     let protocols: String
     let paging: Language.PagingNames?
+    let globalParameters: [ParameterViewModel]
+    // A dictionary of all the named operation group. Key is the group name , Value is the operation group view model.
+    let namedOperationGroups: [String: OperationGroupViewModel]
+    // A key,Value pairs of all the named operation group for stencil template engine
+    let namedOperationGroupProperties: [KeyValueViewModel]
 
     init(from model: CodeModel) {
         self.name = "\(model.packageName)Client"
         self.comment = ViewModelComment(from: model.description)
-        var items = [OperationGroupViewModel]()
+        var operationGroups = [OperationGroupViewModel]()
+        var namedOperationGroups = [String: OperationGroupViewModel]()
+        var namedOperationGroupProperties = [KeyValueViewModel]()
         for group in model.operationGroups {
-            items.append(OperationGroupViewModel(from: group, with: model))
+            var viewModel = OperationGroupViewModel(from: group, with: model)
+            var groupName = group.name.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
We could build this normalizing logic into the getter so that any time you get `group.name` you always get the trimmed version.

Reply:
Changed per feedback.